### DISCUSSION TITLE
Add "hacktoberfest" topic to Hono's repositories

### DISCUSSION THREAD

**Discussion by rafaell-lycan** [honojs/hono#1533] [General]
Hey 👋 

What about adding "hacktoberfest" topic on Hono's repositories to participate and attract contributors?
https://hacktoberfest.com/participation/

**Comment by yusukebe** [comment]
Hi!

I've already added the topic on there:

* [honojs/hono: Lightweight, Ultrafast, Web Standards](https://github.com/honojs/hono)
* [honojs/middleware: monorepo for Hono third-party middleware/helpers/wrappers](https://github.com/honojs/middleware)
* [honojs/website: Repository for hono.dev](https://github.com/honojs/website)
